embedded/Sentinel Embedded/platformio.ini

```diff
@@ -0,0 +1,34 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env]
+platform = espressif32
+board = esp-wrover-kit
+framework = arduino
+monitor_speed = 115200
+upload_port = /dev/cu.wchusbserial54350352381
```

Comment:
My own upload port, this would be different for others using it.

Reply:
Might be a dumb question, I'm not super familiar with C++, but would it be possible to set this with an environment variable, or is that too much of a hassle?

Reply:
Its a platformIO sepcific environment you can just change in VSCode, takes like 1 second - a button click.